test/release/examples/primers/modules.chpl

```diff
@@ -0,0 +1,390 @@
+/*
+ * Module Primer
+ *
+ * This primer introduces the concept of modules, a strategy for encapsulating
+ * code for later use.  It covers:
```

Comment:
"strategy" -> "concept"
"later use" -> "use" or "use by other code" (or just drop "for later use"?)